common/messages/messages_en.go

```diff
@@ -211,6 +211,55 @@ var messagesEn = map[string]util.Message{
 		Type:        util.DeletionFailed,
 		Action:      "User need to review all mount targets and delete them first before deleting file share. Run `ibmcloud is share-targets <SHARE_ID>` to get the mount target for the file share. ",
 	},
+	"SnapshotSpaceOrderFailed": {
+		Code:        "SnapshotSpaceOrderFailed",
+		Description: "Snapshot space order failed for the given volume ID",
+		Type:        util.ProvisioningFailed,
+		RC:          500,
+		Action:      "Please check your input",
+	},
+	"ListSnapshotsFailed": {
+		Code:        "ListSnapshotsFailed",
+		Description: "Unable to fetch list of snapshots.",
+		Type:        util.RetrivalFailed,
+		RC:          404,
+		Action:      "Run 'ibmcloud is share-snapshots' to list available snapshots in your account.",
```

Comment:
This need to be improved, from this error and action user will not be able to validate.

Reply:
hmm , the ibmcloud is share-snapshots is also valid but it will list all snapshots for all share-ids. Ideally the command you specified makes sense in our case as we wont support ListSnapshots for all shares it will only be base on share-id based.